chargeOclock/Source/Models/Client.swift

```diff
@@ -15,10 +15,15 @@ protocol ClientSettable {
 class Client {
 
 	var id: Int = 0
-	var name: String = ""
+	var isDirty: Bool = false
 
-	init(id: Int, name: String) {
-		self.id = id
+	var name: String = "" {
+		didSet {
+			isDirty = true
+		}
+	}
+
+	init(name: String) {
 		self.name = name
```

Comment:
I'm thinking of also removing the `isDirty` parameter. Except if I'm actually getting it wrong.
Do you agree for the `isDirty` parameter to be removed?